### PR TITLE
DENG-1781- Remove urlbar_events_temp_v2 view and repoint urlbar_events view to v2

### PR DESCRIPTION
**Pull request by alekhyamoz**
As a requirement for https://mozilla-hub.atlassian.net/browse/DENG-1781
With this PR,  
1. the `urlbar_events` view will point to the `v2` table
2. The original query for v1 is replaced by the query for v2 in `sql_generator`
3. DAG is regenerated to remove `v1` from getting populated.

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-1869)

### CODE REVIEW

File: dags/bqetl_urlbar.py [hunk 1]
```diff
@@ -48,27 +48,9 @@
...
-            "telemetry-alerts@mozilla.com",
-        ],
-        date_partition_parameter="submission_date",
-        depends_on_past=False,
-    )
-
     firefox_desktop_urlbar_events__v2 = bigquery_etl_query(
         task_id="firefox_desktop_urlbar_events__v2",
-        destination_table="urlbar_events_v2",
+        destination_table="urlbar_events_v1",
```
Comment (curtismorales) [inline dags/bqetl_urlbar.py:53]: Is this supposed to build v1?

File: sql_generators/urlbar_events/templates/desktop_query.sql [hunk 2]
```diff
@@ -1,190 +1,110 @@
...
+CREATE TEMP FUNCTION get_event_action(event_name STRING, engagement_type STRING) AS (
+  CASE
+    WHEN event_name = 'engagement'
+      AND (engagement_type IN ("click", "drop_go", "enter", "go_button", "paste_go"))
+      THEN 'engaged'
+    WHEN event_name = 'abandonment'
+      THEN 'abandoned'
+    WHEN event_name = 'engagement'
+      AND (engagement_type NOT IN ("click", "drop_go", "enter", "go_button", "paste_go"))
+      THEN "annoyance"
```
Comment (curtismorales) [inline sql_generators/urlbar_events/templates/desktop_query.sql:25]: Maybe rearrange these for clarity?

```suggestion
    WHEN event_name = 'engagement'
      AND (engagement_type NOT IN ("click", "drop_go", "enter", "go_button", "paste_go"))
      THEN "annoyance"
    WHEN event_name = 'abandonment'
      THEN 'abandoned'
```

File: sql_generators/urlbar_events/templates/desktop_query.sql [hunk 3]
```diff
@@ -1,190 +1,110 @@
...
+);
+
 WITH events_unnested AS (
   SELECT
     DATE(submission_timestamp) AS submission_date,
+    client_info.client_id AS glean_client_id,
+    metrics.uuid.legacy_telemetry_client_id AS legacy_telemetry_client_id,
     sample_id,
-    client_info.client_id,
+    name AS event_name,
```
Comment (curtismorales) [inline sql_generators/urlbar_events/templates/desktop_query.sql:57]: Can we add the `event` alias for fields that are pulled from there? This one is one, right (`event.name AS event_name`)?

File: dags/bqetl_urlbar.py (hunk 1 above)
Comment (alekhyamoz) [reply dags/bqetl_urlbar.py:53]: we are bumping the version of the table.going forward it will e `v2`

Comment (alekhyamoz) [reply dags/bqetl_urlbar.py]: Fixed it. Thank you for point that out

**Review by rebecca-burwei — COMMENTED** [review]
Hi @alekhyamoz - Could you have another DE review this? I don't understand the infrastructure, so I can't provide any meaningful review.

When we increase the version on a derived table, do we keep the previous version around? In BQ, I see clients_first_seen has both V1 and V2.

**Comment by alekhyamoz** [comment]
> Hi @alekhyamoz - Could you have another DE review this? I don't understand the infrastructure, so I can't provide any meaningful review.
> 
> When we increase the version on a derived table, do we keep the previous version around? In BQ, I see clients_first_seen has both V1 and V2.

In BQ we will have the v1 but we will not be populating it (i.e. remove it from the bqetl repo)

**Review by curtismorales — APPROVED** [review]

**Pull request merged**